hack/build/docker/builder/Dockerfile

```
@@ -52,7 +52,19 @@ COPY output-bazel-arch.sh /output-bazel-arch.sh
 
 RUN curl -L -o /usr/bin/bazel https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-linux-$(sh /output-bazel-arch.sh) && chmod u+x /usr/bin/bazel
 
-# Until we use a ver including the fix for this Bazel issue:
+
+RUN if test "${ARCH}" = x86_64; then \
```

I don't think the environment is kept between build steps and besides, the script is never sourced anyway.
What i would propose then is to replace the Variable with the script.
```suggestion
RUN if test "$(sh /output-bazel-arch.sh)" = x86_64; then \
```

---

Isn't `x86_64` a typo?